package.json

```diff
@@ -32,7 +32,7 @@
   },
   "homepage": "https://github.com/nimiq/keyguard-next#readme",
   "devDependencies": {
-    "@nimiq/core-types": "^1.0.0",
+    "@nimiq/core": "nimiq-network/core",
```

Comment:
The new core version that includes types is not yet released, but available in `master`, thus I'm using the Github path here, and not the NPM version.

Reply:
As soon as the new version including the V3 encryption is released as stable, I will update all CDN links and the package version accordingly. This PR will not be merged before that.